documentation/slate/source/includes/realtime.md

```diff
@@ -22,6 +22,23 @@ For each one of these effects, here's how the Navitia responses will be affected
 A disruption is present in the response of the endpoints described if the request is made during its application period.
 </aside>
 
+
+## <a name="PT_object_collections_data_freshness"></a>Public transport object collections
+
+Several public transport objects have separate collections for `base_schedule` and `realtime`.
+So the data_freshness parameter may affect the number of objects returned depanding on the request.
+
+For exemple when looking for a specific circulation with the collection vehicle_journey with the request like
```

Comment:
```suggestion
For exemple when looking for a specific circulation with the collection vehicle_journey with the request:
```

Reply:
Minor: We can avoid two with: -- with the collection vehicle_journey with the request ...